frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -43,7 +43,6 @@ import { valuesToApiFormat, ViewOnlyBuildingForm } from '../SidebarContents/Buil
 import { getInitialValues, ISearchFields, ViewOnlyLandForm } from '../SidebarContents/LandForm';
 
 interface IMapSideBarContainerProps {
-  refreshParcels: Function;
```

Comment:
Removed this prop as it can't be reached by mapRef from Map.tsx and the function didn't seem to be used.